### PR TITLE
bot-conversation sample migration

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Comment by Prithvi-MSFT** [comment]
Build status  - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5445213719

File: samples/bot-conversation/nodejs/teamsapp.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
+environmentFolderPath: ./env
+projectId: 6de929b1-f6b8-4222-9921-6d07757c5eb4
```
Comment (formulahendry) [inline samples/bot-conversation/nodejs/teamsapp.yml:7]: remove projectId

Comment (Prithvi-MSFT) [reply samples/bot-conversation/nodejs/teamsapp.yml]: removed

**Review by formulahendry — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5452011134

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**